packages/protocol/migrationsConfig.js

```diff
@@ -84,14 +85,14 @@ const DefaultConfig = {
     frozen: true,
   },
   governance: {
-    queueExpiry: WEEK, // Change to 4 weeks once mainnet activated
-    dequeueFrequency: MINUTE, // Change to 1 week once mainnet activated
-    concurrentProposals: 3, // Change to 10 once mainnet activated
-    approvalStageDuration: 30 * MINUTE, // Change to 3 days once mainnet activated
-    referendumStageDuration: HOUR, // Change to 1 week once mainnet activated
-    executionStageDuration: WEEK,
+    queueExpiry: 4 * WEEK,
+    dequeueFrequency: DAY, // Change to 1 week once mainnet activated
+    concurrentProposals: 3,
+    approvalStageDuration: DAY, // Change to 3 days once mainnet activated
+    referendumStageDuration: 2 * DAY, // Change to 1 week once mainnet activated
+    executionStageDuration: 3 * DAY,
     minDeposit: 100, // 100 cGLD
-    participationBaseline: 8 / 10,
+    participationBaseline: 0, // Start with no participation requirements, let the protocol adjust
```

Comment:
do you need to update the participationBaselineFloor as well?  looks like it will get set to the floor otherwise

https://github.com/celo-org/celo-monorepo/blob/8df0872d6b4f9468f0422a8e3fcfae9e7bf64f41/packages/protocol/contracts/governance/Governance.sol#L1159